lxd/daemon.go

```diff
@@ -949,6 +1006,73 @@ func (d *Daemon) Stop() error {
 	return err
 }
 
+// Setup the API web server
+func (d *Daemon) setupWebServer() {
+	/* Setup the web server */
+	d.mux = mux.NewRouter()
+	d.mux.StrictSlash(false)
+
+	d.mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		SyncResponse(true, []string{"/1.0"}).Render(w)
+	})
+
+	for _, c := range api10 {
+		d.createCmd("1.0", c)
+	}
+
+	for _, c := range apiInternal {
+		d.createCmd("internal", c)
+	}
+
+	d.mux.NotFoundHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		logger.Info("Sending top level 404", log.Ctx{"url": r.URL})
+		w.Header().Set("Content-Type", "application/json")
+		NotFound.Render(w)
+	})
+
+}
+
+// Setup external authentication
+func (d *Daemon) setupExternalAuthentication(authEndpoint string) error {
+	if authEndpoint == "" {
+		d.externalAuth = nil
+		logger.Infof("External authentication disabled")
```

Comment:
Why is this being logged?

Reply:
Switching authentication type seem something worth logging

Reply:
It makes sense to print something on startup that states whether external authentication is enabled or not but I don't think it makes sense to print it when the config changes.

If we thought that server configuration changes were worth logging, we'd log all of them.